docs/api-docs/storefront/graphql/graphql-site-content.mdx

```diff
@@ -0,0 +1,1004 @@
+# Site Content with the GraphQL Storefront API
+
+BigCommerce's GraphQL Storefront API allows you to retrieve site content from each storefront channel. Having access to site content makes it easier to build headless storefront applications using BigCommerce's built-in content, such as theme widgets.
+
+Developers can use frontend GraphQL to perform the following: 
+
+- Query [web page content]((https://support.bigcommerce.com/s/article/Web-Pages?language=en_US)), including plain-text pages, contact form pages, blog posts, and more
+- Retrieve [banner content](https://support.bigcommerce.com/s/article/Creating-Editing-Banners?language=en_US) on home, search, category, and brand pages 
+- Retrieve rendered [widgets](https://support.bigcommerce.com/s/article/Page-Builder?language=en_US#builder) built using Page Builder
+- Use the web page URL to retrieve web pages and blog posts
+
+For more info on web pages, see [Page Types](https://developer.bigcommerce.com/docs/rest-content/pages#page-types). Queries don't support RSS-syndicated content pages, as these are deprecated.
+
+## Web pages
+
+Queries for web pages retrieve web pages for a specific storefront. The request URL contains the storefront domain. For example, to retrieve web pages from the `yourstore.example.com` storefront domain, send a request to the URL `https://{yourstore.example.com}/graphql`. If the request URL uses the permanent store URL, the query returns web pages from the default storefront channel.
```

Comment:
I think this should be rewritten for clarity. Keep in mind that most headless storefronts use channel canonical URLs.

I would try to convey:
- Storefront GraphQL queries are in the context of a particular Channel, and will only return web pages related to that channel ID
- We don't need to mention the different URLs here, maybe we can just link here to help people understand which URL to use: https://developer.bigcommerce.com/api-docs/storefront/graphql/graphql-api-overview#faq-querying-for-channels-other-than-stencil-storefronts-and-querying-from-external-systems

Reply:
✔️  rewritten & used link instead